src/libpriv/rpmostree-postprocess.c

```diff
@@ -594,7 +594,7 @@ convert_var_to_tmpfiles_d (int            rootfs_dfd,
 {
   GLNX_AUTO_PREFIX_ERROR ("Converting /var to tmpfiles.d", error);
 
-  g_autoptr(RpmOstreePasswdDB) pwdb = rpmostree_passwddb_open (rootfs_dfd, cancellable, error);
```

Comment:
Note: the `cancellable` was not really used even in the C logic, so I just dropped it here.